css/properties/font-variant-caps.json

```
@@ -57,10 +57,10 @@
               "version_added": "41"
             },
             "safari": {
-              "version_added": false
+              "version_added": "9.1"
             },
             "safari_ios": {
-              "version_added": false
+              "version_added": "10"
```

```suggestion
              "version_added": "9.3"
```

This is what I'd expect since the point releases of Safari and iOS go together even if the numbers are different. This has the same release date and WebKit version according to `safari_ios.json`.

But more importantly, I could confirm this using https://mdn-bcd-collector.appspot.com/tests/css/properties/font-variant-caps on the iOS 9.3 simulator on BrowserStack.